### ISSUE TITLE
Doesn't fix files via SSH (via SSH File Systems)

### ISSUE THREAD

**Issue by KEINOS** [junstyle/vscode-php-cs-fixer#77]
The fixer works like a charm with the files on local storage in my Mac. But not with the files via SSH (files which opened with [SSH File Systems](https://marketplace.visualstudio.com/items?itemName=Kelvin.vscode-sshfs) extension).

### Steps to produce

1. Open VSCode.
2. SSH to any Linux/Unix server using [SSH-FS](https://marketplace.visualstudio.com/items?itemName=Kelvin.vscode-sshfs).
3. Create a php file as below:

```php
#!/usr/bin/env php
<?php

function foo(){
    echo 'bar';
}

foo();

```

4. Press "`alt`+`shift`+`F`" or choose "Format Document" from the right mouse context menu. (Rule = PSR-2)

#### The output

```php
#!/usr/bin/env php
<?php

function foo()
{
    echo 'bar';
}

foo();

```

#### Extension log

- `cmd-shift-p` -> Search `Show Logs` -> `Extension Host`
    - Empty

#### The output

- Nothing changes.

#### Extension log

- `cmd-shift-p` -> Search `Show Logs` -> `Extension Host`

```text
[2019-04-13 12:16:08.527] [exthost] [error] [junstyle.php-cs-fixer] provider FAILED
[2019-04-13 12:16:08.527] [exthost] [error] undefined
```

<details><summary>Results of Toggled Developer Tools</summary>

```text
[Extension Host]% (5) [
  "fix", "--using-cache=no",
  "/var/folders/k1/ld1jgsbs149bhxr9yqz_c74r0000gn/T/analyze.php", 
  "--rules=@PSR2",
  "--path-mode=override"
]
  0: "fix"
  1: "--using-cache=no"
  2: "/var/folders/k1/ld1jgsbs149bhxr9yqz_c74r0000gn/T/analyze.php"
  3: "--rules=@PSR2"
  4: "--path-mode=override"length:
  5__proto__: Array(0)
    concat: ƒ concat()
    constructor: ƒ Array()
    copyWithin: ƒ copyWithin()
    entries: ƒ entries()
    every: ƒ every()
    fill: ƒ fill()
    filter: ƒ filter()
    find: ƒ find()
    findIndex: ƒ findIndex()
    forEach: ƒ forEach()
    includes: ƒ includes()
    indexOf: ƒ indexOf()
    join: ƒ join()
    keys: ƒ keys()
    lastIndexOf: ƒ lastIndexOf()
    length: 0
    map: ƒ map()
    pop: ƒ pop()
    push: ƒ push()
    reduce: ƒ reduce()
    reduceRight: ƒ reduceRight()
    reverse: ƒ reverse()
    shift: ƒ shift()
    slice: ƒ slice()
    some: ƒ some()
    sort: ƒ sort()
    splice: ƒ splice()
    toLocaleString: ƒ toLocaleString()
    toString: ƒ toString()
    unshift: ƒ unshift()
    values: ƒ values()
    Symbol(Symbol.iterator): ƒ values()
    Symbol(Symbol.unscopables): {
      copyWithin: true,
      entries: true,
      fill: true,
      find: true, findIndex: true, …
    }
    __proto__: Object

console.ts:134 [Extension Host] Loaded config default.
console.ts:134 [Extension Host] 
Fixed all files in 0.002 seconds, 10.000 MB memory used
console.ts:134 [Extension Host] 
Files that were not fixed due to errors reported during linting before fixing:
console.ts:134 [Extension Host]    1) /private/var/folders/k1/ld1jgsbs149bhxr9yqz_c74r0000gn/T/analyze.php
```

</details>

### Env info

- macOS Mojave (OSX 10.14.4)
- Visual Studio v1.33.1
- junstyle.php-cs-fixer v0.1.118
- kelvin.vscode-sshfs v1.14.7

<details><summary>Version details</summary>

```shellsession
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.14.4
BuildVersion:	18E226
$ php --version
PHP 7.1.23 (cli) (built: Feb 22 2019 22:08:13) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
$ phpcs --version
PHP_CodeSniffer version 3.4.1 (stable) by Squiz (http://www.squiz.net)
$ php-cs-fixer --version
PHP CS Fixer 2.14.0 Sunrise by Fabien Potencier and Dariusz Ruminski (b788ea0)
$ ls ~/.vscode/extensions/ | grep php-cs-fixer
junstyle.php-cs-fixer-0.1.118
$ ls ~/.vscode/extensions/ | grep sshfs
kelvin.vscode-sshfs-1.14.7
$ code --version
1.33.1
51b0b28134d51361cf996d2f0a1c698247aeabd8
x64
$ code --list-extensions
bierner.markdown-preview-github-styles
DavidAnson.vscode-markdownlint
felixfbecker.php-debug
felixfbecker.php-intellisense
felixfbecker.php-pack
jetmartin.bats
junstyle.php-cs-fixer
Kelvin.vscode-sshfs
marcostazi.VS-code-vagrantfile
mkloubert.vscode-remote-workspace
MS-CEINTL.vscode-language-pack-ja
ms-python.python
ms-vsliveshare.vsliveshare
PeterJausovec.vscode-docker
rexshi.phpdoc-comment-vscode-plugin
```

</details>

**Comment by junstyle** [comment]
nothing change?
it has changed, you look carefully

**Comment by KEINOS** [comment]
Thank you for the quick response.

>> #### The output
>> - Nothing changes.
>
> nothing change?

Sorry for the lack of info. The actual behavior output was as below:

### Actual behavior
#### The output

```php
#!/usr/bin/env php
<?php

function foo(){
    echo 'bar';
}

foo();
```

Which the "`{`" brace seem to stay the same, and it wasn't what I expected with the PSR-2 rule.

**Comment by KEINOS** [comment]
It works fine with the local files though. It happens only if I open a file via SSH-FS and run the fixer.

**Comment by junstyle** [comment]
what's the file path of the ssh file?

**Comment by KEINOS** [comment]
Here's the copy of the path from VSCode:

```
/sample/foo.php
```

**Comment by KEINOS** [comment]
I am sorry! My bad !!!

I re-created the workspace and the SSH connection, then rebooted the VSCode and worked!

Here're the steps I took:
1. Closed all the files(tabs) at the VSCode
2. Re-created the workspace
3. Re-declared the SSH connection settings (for SSH-FS)
4. Re-open VSCode

I could open/delete/edit the files, but with the steps above it fixed. So, somehow the workspace and/or SSH settings or the cash seemed to be doing something bad!
This is my bad! I should I tested in a clean environment, sorry.
